java/daikon/Chicory.java

```diff
@@ -113,6 +113,12 @@
   @Option("Number of calls after which sampling will begin")
   public static int sample_start = 0;
 
+  @Option("Should Exception Handling be taken care of?")
```

Comment:
Please make the documentation of this option more explicit.  "taken care of" won't be clear to readers.